specification/workloadmonitor/resource-manager/readme.md

```diff
@@ -75,35 +75,6 @@ csharp:
   clear-output-folder: true
 ```
 
-
-## Python
```

Comment:
@vivek-microsoft Do not pull out Python generation because the python SDK generation is breaking. Most likely cause of the break is generally in swagger.